backend/app/assets/javascripts/spree/backend/taxon_autocomplete.js

```diff
@@ -1,30 +1,20 @@
-'use strict';
+$.fn.taxonAutocomplete = function () {
+  'use strict';
 
-var set_taxon_select = function(){
-  function formatTaxon(taxon) {
-    return Select2.util.escapeMarkup(taxon.pretty_name);
-  }
-
-  if ($('#product_taxon_ids').length > 0) {
-    $('#product_taxon_ids').select2({
+  this.select2({
       placeholder: Spree.translations.taxon_placeholder,
       multiple: true,
       initSelection: function (element, callback) {
-        Spree.ajax({
-          type: "GET",
-          url: Spree.routes.taxons_search,
-          data: {
-            ids: element.val(),
-            without_children: true
-          },
-          success: function (data) {
-            callback(data['taxons']);
-          }
+        var url = Spree.url(Spree.routes.taxons_search, {
```

Comment:
I didn't change either one on purpose, I just copied it from... somewhere. Weird. Were those things _just_ changed recently?  I may have forked for this PR before the change?  I'm super confused, I don't understand that code enough to have changed it intentionally, I copied it from somewhere, I didn't mean to change it. 

I don't entirely understand what's going on, but I can change it however you like.